packages/db/src/lib/execution.ts

```diff
@@ -178,7 +178,9 @@ export const getWorkflowNameIntersectFromGranuleIds = async (
   const numberOfGranules = granuleCumulusIdsArray.length;
   const { executions: executionsTable, granulesExecutions: granulesExecutionsTable } = TableNames;
 
-  const aggregatedWorkflowCounts = await knexOrTransaction(executionsTable)
+  const aggregatedWorkflowCounts: Array<{ workflow_name: string }> = await knexOrTransaction(
```

Comment:
This explicit typcasting concerns me (knex/table could change) ,but I grok why it's needed here :\